packages/erc3k/contracts/ERC3000Registry.sol

```diff
@@ -4,36 +4,24 @@
 
 pragma solidity 0.6.8;
 
-import "@aragon/govern-contract-utils/contracts/erc165/ERC165.sol";
-
 import "./ERC3000.sol";
 import "./ERC3000Executor.sol";
 
-contract ERC3000Registry is ERC3000Interface {
-    mapping (string => bool) public nameUsed;
-
-    event Registered(ERC3000Executor indexed dao, ERC3000 queue, address indexed registrant, string name);
-    event SetMetadata(ERC3000Executor indexed dao, bytes metadata);
-
-    function register(ERC3000Executor _dao, ERC3000 _queue, string calldata _name, bytes calldata _initialMetadata) external
-    {
-        require(!nameUsed[_name], "registry: name used");
-        require(ERC165(address(_queue)).supportsInterface(ERC3000_INTERFACE_ID), "registry: bad interface queue");
-
-        // all will revert if `_dao` is not interface compliant in _setMetadata
-        nameUsed[_name] = true;
-
-        emit Registered(_dao, _queue, msg.sender, _name);
-        _setMetadata(_dao, _initialMetadata);
-    }
-
-    function setMetadata(bytes memory _metadata) public {
-        _setMetadata(ERC3000Executor(msg.sender), _metadata);
-    }
-
-    function _setMetadata(ERC3000Executor _dao, bytes memory _metadata) internal {
-        require(ERC165(address(_dao)).supportsInterface(_dao.exec.selector), "registry: bad interface dao");
-
-        emit SetMetadata(_dao, _metadata);
-    }
+abstract contract ERC3000Registry is ERC3000Interface {
```

Comment:
```suggestion
abstract contract IERC3000Registry is ERC3000Interface {
```

Reply:
Should I rename all contracts to ``IERC3000*`` for having a consistency naming?